dts/arm/st/l4/stm32l432.dtsi

```diff
@@ -59,6 +59,16 @@
 			sample-point = <875>;
 		};
 
+		sdmmc1: sdmmc@40012800 {
```

Comment:
Could you please check RM0394 : it seems that  stm32l432x has none : 
 _SDMMC 
 Not available on STM32L42xxx, STM32L432xx and STM32L442xx devices.
 Not available on STM32L41xxx and STM32L42xxx devices._